include/nbl/builtin/hlsl/type_traits.hlsl

```diff
@@ -77,33 +91,33 @@
     template<class R, class Fn, class... ArgTypes> struct is_nothrow_invocable_r;
  
   // const-volatile modifications
-  template<class T> struct remove_const; (TODO)
-  template<class T> struct remove_volatile; (TODO)
-  template<class T> struct remove_cv; (TODO)
-  template<class T> struct add_const; (TODO)
-  template<class T> struct add_volatile; (TODO)
-  template<class T> struct add_cv; (TODO)
+  template<class T> struct remove_const; (DONE)
+  template<class T> struct remove_volatile; (DONE)
+  template<class T> struct remove_cv; (DONE)
+  template<class T> struct add_const; (DONE)
+  template<class T> struct add_volatile; (DONE)
+  template<class T> struct add_cv; (DONE)
 
   // reference modifications
-  template<class T> struct remove_reference; (TODO)
-  template<class T> struct add_lvalue_reference; (TODO)
-  template<class T> struct add_rvalue_reference; (TODO)
+  template<class T> struct remove_reference; (DONE)
```

Comment:
by symmetry also `DONT USE` `remove_reference` (move it to `impl::`)

Reply:
i think this is safe to use though